build_web_compilers/lib/builders.dart

```diff
@@ -3,25 +3,40 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:build/build.dart';
-
 import 'package:build_modules/build_modules.dart';
-import 'package:build_web_compilers/build_web_compilers.dart';
-
 import 'package:path/path.dart' as p;
 
-Builder devCompilerBuilder(_) => DevCompilerBuilder(useKernel: true);
+import 'build_web_compilers.dart';
+import 'src/platforms.dart';
+
+/// Shared entrypoint builder
 Builder webEntrypointBuilder(BuilderOptions options) =>
     WebEntrypointBuilder.fromOptions(options, useKernel: true);
-PostProcessBuilder dart2JsArchiveExtractor(BuilderOptions options) =>
-    Dart2JsArchiveExtractor.fromOptions(options);
-PostProcessBuilder dartSourceCleanup(BuilderOptions options) =>
-    (options.config['enabled'] as bool ?? false)
-        ? const FileDeletingBuilder(['.dart', '.js.map'])
-        : const FileDeletingBuilder(['.dart', '.js.map'], isEnabled: false);
 
+/// Ddc related builders
+Builder ddcMetaModuleBuilder(BuilderOptions options) =>
+    MetaModuleBuilder.forOptions(ddcPlatform, options);
+Builder ddcMetaModuleCleanBuilder(_) => MetaModuleCleanBuilder(ddcPlatform);
+Builder ddcModuleBuilder([_]) => ModuleBuilder(ddcPlatform);
+Builder ddcBuilder([_]) => DevCompilerBuilder(useKernel: true);
 const ddcKernelExtension = '.ddc.dill';
-Builder ddcKernelBuilder(_) => KernelBuilder(
+Builder ddcKernelBuilder([_]) => KernelBuilder(
     summaryOnly: true,
     sdkKernelPath: p.url.join('lib', '_internal', 'ddc_sdk.dill'),
     outputExtension: ddcKernelExtension,
-    platform: DartPlatform.dartdevc);
+    platform: ddcPlatform);
+
+/// Dart2js related builders
```

Comment:
[nit] Don't write doc comments that apply to multiple things, make it a block comment or remove it.